src/datasource.ts

```diff
@@ -121,6 +121,7 @@ export default class VividCortexDatasource {
       samplesize: calculateSampleSize(from, until, dataPoints),
       until: until,
       host: null,
+      separateHosts: target.separateHosts ? '1' : '0',
```

Comment:
Why are the values to the `separateHosts` strings? According to the [spec](https://github.com/VividCortex/api-metrics#get) they are integers. Might make no difference though.

Reply:
Cool, I didn't know the spec existed, that's really helpful. 
Even though that, it wouldn't make any difference because it's sent as a query parameter in the URL.